mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -53,6 +65,96 @@ class FPGrowthModel[Item: ClassTag] @Since("1.3.0") (
     val associationRules = new AssociationRules(confidence)
     associationRules.run(freqItemsets)
   }
+
+  override def save(sc: SparkContext, path: String): Unit = {
```

Comment:
Add Since tags for this and other overridden methods (though I need to check to make sure this appears correctly in the docs)

Override documentation for save() to state that it only works for Item datatypes supported by DataFrames.